### ISSUE TITLE
When specifying TemplateAlias, the API complains about missing TemplateId

### ISSUE THREAD

**Issue by saasen** [wildbit/postmark-dotnet#66]
Hi.

Firstly, I love this API. It's really neat and definitely simplifies SMTP communication.

Our setup:
```C#
.NET Core 2.1
<PackageReference Include="Postmark" Version="4.0.0" />
```

We are struggling with the API when using `TemplateAlias`. Here's the error that we get:

```C#
PostmarkDotNet.Exceptions.PostmarkValidationException: The 'TemplateId' associated with this request is not valid or was not found.
   at PostmarkDotNet.PostmarkClientBase.ProcessRequestAsync[TRequestBody,TResponse](String apiPath, HttpMethod verb, TRequestBody message)
   at PostmarkDotNet.PostmarkClient.SendEmailWithTemplateAsync(TemplatedPostmarkMessage emailToSend)
   at PostmarkDotNet.PostmarkClient.SendMessageAsync(TemplatedPostmarkMessage emailToSend)
```

Here's our code:

```C#
var message = new TemplatedPostmarkMessage
{
    To = request.Email,
    From = _options.From,
    Bcc = _options.Bcc,
    TemplateAlias = _options.Customer.TemplateAlias,
    TemplateModel = new
    {
        first_name = request.FirstName,
        LastName = request.LastName,
        Voucher = voucher,
        CompanyName = "<company>",
        CompanyAddress = "<address>",
    }
};

await _postmarkClient.SendMessageAsync(message);
```

I've double checked that the `TemplateAlias` value is set and to a template that exists. It still complaints about a missing `TemplateId`, but isn't the point of having an alias to skip the ID?

**Comment by atheken** [comment]
Hi, this *should* be working, do you mind going on over to [Postmark Support](https://postmarkapp.com/contact), sharing your template id and alias there and we can troubleshoot it from there?

**Comment by atheken** [comment]
This was a bug in the client. Sorry for the runaround, but we've patched it and released it as 4.0.1 -- Thanks for bringing it to our attention, and please let us know if you have any more questions.
